src/main/java/com/wavefront/sdk/common/clients/WavefrontClient.java

```diff
@@ -118,7 +118,7 @@
      * @param server A server URL of the form "https://clusterName.wavefront.com" or "http://internal.proxy.com:port"
      * @param token  A valid API token with direct ingestion permissions
      */
-    protected Builder(String server, @Nullable String token) {
+    public Builder(String server, @Nullable String token) {
```

Comment:
Can we update `Main` so that it uses this infrastructure as well or is that a bad idea?

Reply:
We would be better suited to update the README.md

Reply:
@MikeMcMahon for this, can you update the README.md to reflect the recent changes?